.drone.yml

```diff
@@ -0,0 +1,32 @@
+pipeline:
+
+  build_image:
+    image: docker:1.11
+    environment:
+      - DOCKER_HOST=tcp://127.0.0.1:2375
+    commands:
+      docker build -t symmetricds .
+    when:
+      event: [pull_request, push, tag]
```

Comment:
Could we separate out the push so it only runs on push to master?

Means duplicating the job which is annoying but otherwise drone will do a build for no reason.

Reply:
Or I suppose we could move the `docker build` in to the job below but we don't do that elsewhere.